src/vaults/VaultInternal.ts

```diff
@@ -953,12 +953,12 @@ class VaultInternal {
         [
           ["a.txt", 0, 2, 0], // new, untracked
           ["b.txt", 0, 2, 2], // added, staged
-          ["c.txt", 0, 2, 3], // added, staged, with unstaged changes
+          ["c.txt", 0, 2, 3], // added, staged, with unStaged changes
           ["d.txt", 1, 1, 1], // unmodified
-          ["e.txt", 1, 2, 1], // modified, unstaged
+          ["e.txt", 1, 2, 1], // modified, unStaged
           ["f.txt", 1, 2, 2], // modified, staged
-          ["g.txt", 1, 2, 3], // modified, staged, with unstaged changes
-          ["h.txt", 1, 0, 1], // deleted, unstaged
+          ["g.txt", 1, 2, 3], // modified, staged, with unStaged changes
+          ["h.txt", 1, 0, 1], // deleted, unStaged
           ["i.txt", 1, 0, 0], // deleted, staged
```

Comment:
You might want to align your case sensitivity here.

Also should `Type StatusRow` be an actual type somewhere in `types.ts`?

Reply:
The type is defined by `isomorphic-git` and we don't use it in any form besides consuming it in this one part of the code. It is only named in this comment block to make it clear what we're getting and how to interpret it.

Reply:
No I mean `unStaged` vs `unstaged`.